controllers/get-controller.js

```diff
@@ -1,12 +1,25 @@
-const { fetchTopics } = require("../models/get-models");
+const { fetchTopics, fetchArticles } = require("../models/get-models");
 
 exports.getTopics = (req, res, next) => {
   fetchTopics()
     .then((data) => {
-      res.status(200).send({topics :data});
+      res.status(200).send({ topics: data });
     })
-    .catch((err)=>{
-        console.log(err)
-        next(err)
+    .catch((err) => {
+      console.log(err);
+      next(err);
+    });
+};
+
+exports.getArticles = (req, res, next) => {
+  fetchArticles(req.params)
```

Comment:
dot notation is fine, but you could destructure before this for a nicer argument to pass. This however is subjective, so if you want to leave it you can.